templates/CRM/Core/BillingBlock.js

```diff
@@ -1,71 +1,62 @@
 // http://civicrm.org/licensing
-(function($) {
+(function($, _) {
 
-  /**
-   * Adds the icons of enabled credit cards
-   * Handles clicking on a icon.
-   * Changes the icon depending on the credit card number.
-   * Removes spaces and dashes from credit card numbers.
-   */
-  function civicrm_billingblock_creditcard_helper() {
-    $(function() {
-      $.each(CRM.config.creditCardTypes, function(key, val) {
-        var html = '<a href="#" data-card_type=" + key + " title="' + val + '" class="crm-credit_card_type-icon-' + val.css_key + '"><span>' + val.label + '</span></a>';
```

Comment:
Note `data-card_type` was never set correctly due to bad js concatenation here, so it's obviously unused and not needed.